catalog/gke/cluster/setters.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: setters
+data:
+  # The name of this cluster
+  cluster-name: example-us-west4 # kpt-set: ${cluster-name}
+  # The environment (set as a label on the cluster)
+  environment: dev # kpt-set: ${environment}
+  # The compute location (region or zone)
+  location: us-east4 # kpt-set: ${location}
+  # The private IP range for masters to use when peering to the VPC
+  master-ip-range: 10.254.0.0/28 # kpt-set: ${master-ip-range}
+  # The name of the VPC in which to create this cluster
+  network-name: default # kpt-set: ${network-name}
+  # The project that the VPC is in
+  network-project-id: network-project-id # kpt-set: ${network-project-id}
+  # The namespace in which to reference the subnet resource
+  networking-namespace: networking # kpt-set: ${networking-namespace}
+  # The namespace in which to manage cluster resources
+  platform-namespace: config-control # kpt-set: ${platform-namespace}
+  # The project in which to manage cluster resources
+  platform-project-id: platform-project-id # kpt-set: ${platform-project-id}
+  # The namespace in which to manage service enablement resources
+  projects-namespace: projects # kpt-set: ${projects-namespace}
```

Comment:
We should move shared setters to the top-level package. The way it's currently set up you'll have to update the value in each subpackage.

Reply:
I tested it. It works as-is. 

The inner pipelines run before the outer pipelines, which means inner values only matter for setters that aren't in the outer values.

I filed a FR to provide what I think would be more intuitive behavior: https://github.com/GoogleContainerTools/kpt/issues/2385

Reply:
I see. That's definitely not intuitive but we don't need to block on it.